### PR TITLE
Shorter, more focused referencing guide

### PR DESCRIPTION
**Pull request by handrews**
This reworks the less-controversial parts of a prevoius attempt (#69) at a referencing guide that was over-ambitious, among other things.  It's amazing how being in better health makes my writing less cranky and confrontational! 😅

### CODE REVIEW

File: referencing/faq.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* OAS 3.1 Schema Objects, OAS 3.0 Reference Objects, and both OAS 3.0 and 3.1 Path Item Objects all allow keywords alongsidea`"$ref"`, each with slightly different semantics; generic reference removal tools, particularly ones that predate OAS 3.1, may fail to preserve the behavior while removing references
+* In OAS 3.1 Schema Objects that use `"$id"`, reference removal tools that are not aware of `"$id"` will resolve references impacted by the `"$id"` incorrectly
+* The Link Object's `"operationRef"` cannot be replaced with an inline Operation Object (although if the target Operation Object has a unique-within-the-OAD `"operationId"`, then the `"operationRef"` can be replaced by the `"operationId"`)
+* The Discriminator Object's `"mapping"` field forbids inline Schema Objects; URI-reference values can be replaced with names from the `"schemas"` section of the Components Object, but when the Discriminator Object is in a referenced document, there are ambiguities involvied in how to resolve those names
```
Comment (notEthan) [inline referencing/faq.md:21]: typo 'involvied'

File: referencing/faq.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+* It is possible for a Link Object's `"operationId"` to point to an Operation Object in a Path Item Object that is never referenced; it is unclear how this case should handle it, which is one reason the OAS itself [advises against](https://spec.openapis.org/oas/v3.1.0#fixed-fields-16) using `"operationId"` in  multi-document OADs
+* Security Scheme Objects are never referenced, only named in Security Requirement Objects, and must be taken into account
+* There is no standard place to put referenced Path Item Objects in OAS 3.0
+* The resolution of server configuration and security settings can be unclear in multi-document OADs
+
+Again, in many cases none of this is a problem, and there are [tools](tools.openapis.org) that will do what you want.
```
Comment (notEthan) [inline referencing/faq.md:35]: 'tools' link needs https://

File: referencing/what.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+| 3.x         | Path Item Object | `"$ref"`       | allowed under some circumstances | logically replace the Path Item Object containing the `"$ref"` with a Path Item Object that combines the fields of the target Path Item Object with the non-`"$ref"` fields of the Path Item Object containing the `"$ref"`, as long as none of those fields conflict |
+| 3.1         | Schema Object | `"$ref"`   | allowed | Apply the target Schema Object to the same instance location as the Schema Object containing the `"$ref"`, and combine the results with the results of other keywords in the Schema Object containing the `"$ref"` just as you would any other keyword results; this is more-or-less equivalent to using a one-element `"allOf"` |
+| 3.x         | Link Object   | `"operationRef"` | allowed, except `"operationId"` | treat the reference target as the target of the link described by the Link Object |
+| 3.x         | Discriminator Object | `"mapping"` | n/a | for each name under `"mapping"`, if the value is not the name of a Schema Object under the Components Object, treat it as a reference to the schema to use when the discriminator field matches the mapping name |
```
Comment (notEthan) [inline referencing/what.md:25]: Very informative. Just to check, for the 3.x labels these are all going to be unchanged in 3.2+ until moonwalk?

**Review by notEthan — COMMENTED** [review]
👍

File: referencing/what.md (hunk 3 above)
Comment (handrews) [reply referencing/what.md:25]: @notEthan thanks!  They will remain compatible, more so than 3.0 to 3.1.  I think we might deprecate some things like the special Path Item Object `$ref` behavior, but we won't remove anything.

**Comment by lornajane** [comment]
Thanks @handrews! I need more time to look at this properly, there are some really great nuggets in here of what doesn't work and why, but I think we might also add some "do it this way" advice and some examples to try to lead people out of the dark place in which they find themselves. Would it be okay for me to add changes directly to this pull request to restructure some of the content?

**Comment by handrews** [comment]
I forgot to add that I put the rendered changes up here:

* https://handrews.github.io/renderings/learn-referencing/referencing/faq.html
* https://handrews.github.io/renderings/learn-referencing/referencing/what.html

File: referencing/faq.md (hunk 2 above)
Comment (lornajane) [reply referencing/faq.md]: I got this one in my updates. Thanks @notEthan !

**Comment by lornajane** [comment]
Thanks for being open to some content restructure @handrews - I shifted things out of FAQ format (not considered current practice in documentation, and not an established pattern in these docs) mostly because those articles can get long and difficult to navigate. The content is split into topics, with each page added to the sidebar.

In general, I think paragraphs might be easier to read than bullet points, but I suggest we don't block the pull request for that (I've done enough holding this one up already!).

I would have loved to have seen at least one or two examples in the text, I think that's something that the learn site can excel at - but we can add them later, maybe after hearing user feedback.

**Comment by handrews** [comment]
@lornajane thanks!  I will look at this in detail soon.  Just to respond to a few comments first:

>  shifted things out of FAQ format (not considered current practice in documentation

Great!  I'm not up on current best practices and was not attached to a FAQ format at all.

> In general, I think paragraphs might be easier to read than bullet points

I've been told "bullet points, not paragraphs!" over and over by people who outright refuse to read anything I write in prose because I'm too wordy.  I'm happy for anyone else to restructure it, I'm just hyper-sensitive at this point.

> I would have loved to have seen at least one or two examples in the text

I am horrible at examples. I learn theory-first, unless the theory just isn't articulated and I have to construct it from examples.  Most people learn examples-first.  I've never understood how to connect with that, so hopefully others will fill them in.  I'm happy to provide a topic list or feedback, though.

**Comment by lornajane** [comment]
Let's leave the bullet points and not worry too much about the examples. They can be fleshed out over time - I do think they're useful, but let's add them as we see them in the wild or something!

**Review by lornajane — APPROVED** [review]
Approved, but I'll leave you time to take another look before one of us merges
